libstorage/BinLogHandler.cpp

```diff
@@ -304,19 +304,39 @@ void BinLogHandler::encodeEntries(
                                   << LOG_KV("id", entry->getID())
                                   << LOG_KV("status", (uint32_t)status);*/
         buffer.insert(buffer.end(), (byte*)&status, (byte*)&status + 1);
+        int flagSize = (vecField.size() - 3) / 8 + 1;  // exclude STATUS,NUM_FIELD,ID_FIELD
```

Comment:
binlog version should be changed?

Reply:
binlog hasn't been released yet, so don't need to change the version number

Reply:
binlog hasn't been released yet, so don't need to change the version number